package.json

```diff
@@ -816,10 +816,12 @@
         "vscode-azureappservice": "^0.86.2",
         "vscode-azureextensionui": "^0.48.5",
         "vscode-azurekudu": "^0.2.0",
+        "vscode-dts": "^0.3.1",
```

Comment:
If I understand what this is used for, I think this should be a dev dependency.

For reference I found another Microsoft extension include it as a dev dependency:
https://github.com/microsoft/vscode-simple-jupyter-notebook/blob/77cd8b0f1695e2228756f9ace1d8e2b4d4d21c6c/package.json#L122

Reply:
Yes, that makes more sense